packages/semver/src/executors/version/version.ts

```diff
@@ -34,27 +36,28 @@ export function versionWorkspace({
   skipProjectChangelog: boolean;
   workspaceRoot: string;
 } & CommonVersionOptions) {
-  return concat(
```

Comment:
Using `concat()` like this caused emissions from `getProjectRoots()` and `getPackageFiles()` to get passed out.

By using `concatMap()` and `reduce()` the execution order can remain the same, but the output can just be the rend of the chain.